src/status_im/models/browser.cljs

```diff
@@ -72,7 +73,10 @@
     (assoc (update-dapp-permissions-fx cofx {:dapp        dapp-name
                                              :permissions (vec (set (concat (keys permissions-allowed)
                                                                             user-permissions)))})
-           :send-to-bridge-fx [permissions-allowed webview])))
+           :send-to-bridge-fx [{:type constants/status-api-success
+                                :data permissions-allowed
+                                :keys (keys permissions-allowed)}
+                               webview])))
 
 (defn next-permission [cofx params & [permission permissions-data]]
```

Comment:
models functions should have a fixed number of parameters where the last one is `cofx` in order to be composable with `merge-fx` macro, e.g. in this case `(defn next-permission [params permission permission-data cofx] `